app/controllers/api/base_controller/generic.rb

```diff
@@ -246,9 +246,12 @@ def model_ident(model, type = nil)
       end
 
       def validate_id(id, key_id, klass)
-        if id.nil? || (key_id == "id" && !id.integer?)
-          raise BadRequestError, "Invalid #{klass} #{key_id} #{id || "nil"} specified"
+        id.split(',').map(&:to_i).each do |id|
+          if id.nil? || (key_id == "id" && !id.integer?)
+            raise BadRequestError, "Invalid #{klass} #{key_id} #{id || "nil"} specified"
+          end
         end
+        id.split(/\s*,\s*/)
```

Comment:
Hm this doesn't seem right, we have an existing method for bulk resource actions already that doesn't require comma separated IDs, https://www.manageiq.org/docs/reference/latest/api/overview/bulk_query.html